src/main/java/com/pnu/dev/shpaltaif/controller/PostController.java

```diff
@@ -31,12 +33,18 @@ public String findById(@PathVariable Long id, Model model) {
         return "post/show";
     }
 
-    @GetMapping
+    @GetMapping("/partial")
     public String findAll(@PageableDefault(size = 10, sort = "createdAt", direction = Sort.Direction.DESC)
-                                  Pageable pageable, Model model) {
-
-        Page<Post> posts = postService.findAll(pageable);
-        model.addAttribute("posts", posts);
+                                  Pageable pageable,
+                          PostFiltersDto postFiltersDto,
+                          Model model) {
+
+        try {
```

Comment:
Not the best approach to follow, but at least in won't result in any bugs and not overcomplicates logic of global exception handler.